### PR TITLE
deque vs. stack article

### PR DESCRIPTION
**Pull request by sk1418**

### CODE REVIEW

File: core-java-modules/core-java-collections-4/src/main/java/dequestack/ArrayLifoStack.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package dequestack;
```
Comment (eric-martin) [inline core-java-modules/core-java-collections-4/src/main/java/dequestack/ArrayLifoStack.java:1]: package com.baeldung.collections.dequestack;

File: core-java-modules/core-java-collections-4/src/main/java/dequestack/LifoStack.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package dequestack;
```
Comment (eric-martin) [inline core-java-modules/core-java-collections-4/src/main/java/dequestack/LifoStack.java:1]: package com.baeldung.collections.dequestack;

File: core-java-modules/core-java-collections-4/src/main/java/dequestack/ArrayLifoStack.java (hunk 1 above)
Comment (sk1418) [reply core-java-modules/core-java-collections-4/src/main/java/dequestack/ArrayLifoStack.java:1]: fixed

File: core-java-modules/core-java-collections-4/src/main/java/dequestack/LifoStack.java (hunk 2 above)
Comment (sk1418) [reply core-java-modules/core-java-collections-4/src/main/java/dequestack/LifoStack.java:1]: fixed

**Pull request merged**